documentation/asciidoc/accessories/camera/synchronous_cameras.adoc

```diff
@@ -1,102 +1,97 @@
 == Synchronous Captures
 
-Both the HQ Camera and the Global Shutter Camera, have support for synchronous captures.
-Making use of the XVS pin (Vertical Sync) allows one camera to pulse when a frame capture is initiated.
-The other camera can then listen for this sync pulse, and capture a frame at the same time as the other camera.
+The High Quality (HQ) Camera supports synchronous captures.
+Using the XVS pin (Vertical Sync) allows one camera (the "source") to pulse when a frame capture is initiated.
+Other ("sink") cameras can listen for this pulse, and capture a frame at the same time as the source camera.
 
-=== Using the HQ Camera
+[NOTE]
+====
+Global Shutter (GS) Cameras can also be operated in a synchronous mode. However, the source camera will record one extra frame. Instead, for the GS Camera we recommend using an xref:camera.adoc#external-trigger-on-the-gs-camera[external trigger source]. You cannot synchronise a GS Camera to an HQ Camera.
```

Comment:
People might misinterpret "You cannot synchronise a GS Camera to an HQ Camera." to mean that you _can_ synchronise an HQ Camera to a GS Camera"? :wink:  Perhaps changing "to" to "with" is safer?

Reply:
"and"